pimd/pim_cmd.c

```diff
@@ -8432,14 +8432,18 @@ DEFPY_HIDDEN (interface_ip_igmp_query_generate,
 	      "IGMP version number\n")
 {
 	VTY_DECLVAR_CONTEXT(interface, ifp);
-	int igmp_version = 2;
+	int igmp_version;
```

Comment:
This command is working as documented:

http://docs.frrouting.org/en/latest/pim.html#clicmd-ip-igmp-generate-query-once-version-2-3

Why are we changing it?

Reply:
Updated the document as well. When user does not provide the version in the command, version is taken from the interface.